### PR TITLE
fix(loader): update loader ami with tablet aware driver

### PR DESCRIPTION
**Pull request by soyacz**
Loader ami used old c-s which didn't have java driver with tablets support.

Created new image with c-s from 2024.2.0-rc0 and updated scylla-bench to 0.1.21.

### Testing

- [X] - https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/longevity-5gb-1h-MajorCompactionMonkey-aws-test/15/

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: defaults/aws_config.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ user_credentials_path: '~/.ssh/scylla_test_id_ed25519'
 instance_type_loader: 'c6i.xlarge'
 instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
-ami_id_loader: 'scylla-qa-loader-ami-v20-ubuntu22'
+ami_id_loader: 'scylla-qa-loader-ami-v21-ubuntu22'
```
[the remark this answers is not in the document]
Comment (fruch) [reply defaults/aws_config.yaml:10]: you can get it from argus one a job is running

[the remark this answers is not in the document]
Comment (soyacz) [reply defaults/aws_config.yaml:10]: ugh, wrong params I had set and didn't use it, retrying

**Comment by soyacz** [comment]
I think https://github.com/scylladb/scylla-cluster-tests/issues/7755 is causing issues here as well
cc @dimakr 
Trying without ssl: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/longevity-5gb-1h-MajorCompactionMonkey-aws-test/15/

**Comment by fruch** [comment]
> I think #7755 is causing issues here as well cc @dimakr Trying without ssl: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/longevity-5gb-1h-MajorCompactionMonkey-aws-test/15/

@soyacz are you using 
`use_prepared_loaders` ?

**Comment by soyacz** [comment]
> > I think #7755 is causing issues here as well cc @dimakr Trying without ssl: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/longevity-5gb-1h-MajorCompactionMonkey-aws-test/15/
> 
> @soyacz are you using `use_prepared_loaders` ?
> 
> seems like you used `bare_loaders: true`, which explain why it's broken

yes, used bare loaders (prepared one also needed to be set up, othewise docker was used).
I wanted to avoid `Install java-8 for workaround hdrhistogram` - is it still needed?

**Comment by soyacz** [comment]
https://argus.scylladb.com/workspace?state=WyI1YmQ4ZDJjZi01NzYwLTQwNWUtOGM3MC1mYTBmZDJiYWQ1ZjAiXQ - passed (certs disabled).

**Comment by fruch** [comment]
> > > I think #7755 is causing issues here as well cc @dimakr Trying without ssl: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/longevity-5gb-1h-MajorCompactionMonkey-aws-test/15/
> > 
> > 
> > @soyacz are you using `use_prepared_loaders` ?
> > seems like you used `bare_loaders: true`, which explain why it's broken
> 
> yes, used bare loaders (prepared one also needed to be set up, othewise docker was used). I wanted to avoid `Install java-8 for workaround hdrhistogram` - is it still needed?

I don't know, I guess it's not

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**